gatewayservice/package.json

```diff
@@ -19,6 +19,7 @@
     "homepage": "https://github.com/arquisoft/wiq_en1a#readme",
     "dependencies": {
         "axios": "^1.6.5",
+        "cookie-parser": "^1.4.6",
```

Comment:
We weill leave the library as we may need it furhgter. PAckage management to prevent unused packages, etc will be done further in the project